src/testing/code-debugging.md

```diff
@@ -886,41 +886,45 @@ You can set these flags either by editing the framework code,
 or by importing the module and setting the value in your
 `main()` method, following by a hot restart.
 
-<dl markdown="1">
-<dt markdown="1">[`debugDumpRenderTree()`][]</dt>
-<dd markdown="1">
-  Call this function when not in a layout or repaint
+[`debugDumpRenderTree()`][]</dt>
```

Comment:
Here and in the other cases below, you'll need to the drop the unmatched `</dt>` tags as they show up in the rendered HTML.

```suggestion
[`debugDumpRenderTree()`][]
```